function-memoization/src/main/java/com/tomtom/kotlin/FunctionMemoizationExtensions.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2020-present, TomTom N.V.
```

Comment:
Please copy-paste exactly the same copyright message as in the Uid.kt class. (Normally, we use the Copyright tools from IntelliJ for that which auto-updates the copyright based on copyrights it recognizes.)
I will update the 2020-2021 in Jan :-)

Reply:
Done!